torch/_dynamo/utils.py

```diff
@@ -94,20 +94,22 @@ def profile_wrapper(*args, **kwargs):
 
 frame_phase_timing = collections.OrderedDict()
 
-curr_frame = 0
-
+TOTAL_FRAME_COUNTER = 0
+torch._logging._internal.register_formatter_prefix_fn(lambda: f"{TOTAL_FRAME_COUNTER}: ")
```

Comment:
This is wrong. The prints you get here will print out whatever the frame counter is at the time the log message triggers, not what the frame count of the `frame_state` you are evaluating is!

I don't think there is any way to find this information directly; we must propagate it to the logger manually, similar to how ShapeEnv is doing it.

Reply:
Sorry, I don't understand the destination? I don't want the total frame count, I want the current frame counter at the time log message triggers, right?

Reply:
Oh, I kind of assumed that this would be useful to see how many frames we are on with recompiles factored. I think I misunderstood the desire of what we want here. Yes, if we want to freeze it to the evaluated frame, this is all bad.